jme3-alloc/src/main/java/com/jme3/alloc/util/NativeBufferUtils.java

```diff
@@ -120,4 +115,16 @@ private NativeBufferUtils() {
      * @return a 32-bit or 64-bit integer (depending on the architecture) representing the memory address of the specified buffer
      */
     public static native long getMemoryAdress(final ByteBuffer buffer);
+    
+    private static void loadNativeBinary() {
+        if (!NativeBinaryLoader.isAutoLoad()) {
+            return;
+        }
+        try {
+            /* extracts and loads the system specific library */
+            NativeBinaryLoader.loadLibrary();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Can you please use logger API instead of `System.out.print`?

Reply:
Yeah, sure, which syntax do you suggest? Do you have an example?

Do you think this might be efficient:
```java
Logger.getLogger(NativeBufferUtils.class.getName()).error(e.getMessage());
```

Reply:
I have used a PrintWriter before on JmeSurfaceView and printed the stack trace exception into a Writer, then used the writer data as a String, but I am not going to do this here for now :smile:.

Reply:
```
private static final Logger logger = Logger.getLogger(NativeBufferUtils.class.getName());
```

and use it

```
logger.log(Level.SEVERE, "Failed to load native library", e);
```

Reply:
@Ali-RS Thank you, I am going to defer this enhancement to another PR, we shouldn't do things irrelevant here.

Please, open an issue to remove all regular stdout print statements from the API.

Thanks for your contributions!